Documentation/Troubleshooting/TYPO3.rst

```diff
@@ -104,13 +104,14 @@ change to show errors in the frontend.
 
 .. include:: ../Images/AutomaticScreenshots/DebugSettings/DebugSettings.rst.txt
 
-The following TypoScript setting can also be added the the root template for
+The following TypoScript setting can also be added the the root TypoScript for
 the site to show additional debug information.
 This is particularly useful when debugging Fluid errors:
 
-.. code-block:: typoscript
+..  code-block:: typoscript
+    :caption: config/sites/my_site/setup.typoscript
```

Comment:
Maybe I miss something, but TypoScript in the site configuration? Can you hint me to the changelog/docs?

Reply:
Feature: #103439 - TypoScript provider for sites and sets

https://github.com/TYPO3/typo3/blob/main/typo3/sysext/core/Classes/Configuration/SiteConfiguration.php#L251